azure/table_azure_postgresql_flexible_server.go

```diff
@@ -115,38 +236,32 @@ func tableAzurePostgreSqlFlexibleServer(_ context.Context) *plugin.Table {
 //// LIST FUNCTION
 
 func listPostgreSqlFlexibleServers(ctx context.Context, d *plugin.QueryData, h *plugin.HydrateData) (interface{}, error) {
-	session, err := GetNewSession(ctx, d, "MANAGEMENT")
+	session, err := GetNewSessionUpdated(ctx, d)
```

Comment:
@ParthaI is there any specific logic behind using `GetNewSessionUpdated` naming?

Reply:
@misraved, I don't believe there was any particular reason for this function's naming. I'm not aware of the discussion that led to it, as it was added in [this PR](https://github.com/turbot/steampipe-plugin-azure/pull/747). We can certainly consider changing the function name if needed!

Reply:
Thanks for the confirmation @ParthaI 👍!!

We can raise a separate PR to update the name of the function.